### PR TITLE
(#85) RtNetworks, RtNetwork

### PR DESCRIPTION
**Pull request by llorllale**
This PR is for #85:
* Initial implementations of `Networks` and `Network`
* Left puzzles to continue work

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#157](https://github.com/amihaiemil/docker-java-api/issues/157) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) everybody who has role `REV` is banned at [#157](https://github.com/amihaiemil/docker-java-api/issues/157); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) everybody who has role `REV` is banned at [#157](https://github.com/amihaiemil/docker-java-api/issues/157); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: src/test/java/com/amihaiemil/docker/RtNetworksTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                        + "    \"Name\": \"none\",\n"
+                        + "    \"Id\": \"e086a3893b05ab69242d3c44e49483a3bbbd3a26b46baa8f61ab797c1088d794\",\n"
+                        + "    \"Created\": \"0001-01-01T00:00:00Z\"\n"
+                        + "  },\n"
+                        + "  {\n"
+                        + "    \"Name\": \"host\",\n"
+                        + "    \"Id\": \"13e871235c677f196c4e1ecebb9dc733b9b2d2ab589e30c539efeda84a24215e\",\n"
+                        + "    \"Created\": \"0001-01-01T00:00:00Z\"\n"
+                        + "  }\n"
+                        + "]"
```
[the remark this answers is not in the document]
Comment (llorllale) [reply src/test/java/com/amihaiemil/docker/RtNetworksTest.java]: @amihaiemil I passed a string in order to mirror the docs more closely: basically just copy-pasted the example from the docs (minus irrelevant bits). Still want me to build it?

**Comment by amihaiemil** [comment]
@llorllale I see. Let's leave it like this then.

**Comment by amihaiemil** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:amihaiemil/docker-java-api#157` is not assigned, can't get performer

**Pull request merged**